### PR TITLE
This commit will address Google PR feedback for the native ads PR.

### PR DESCRIPTION
**Pull request by stanley-vungle**
This commit addresses Google's PR feedback for the native ad/COPPA PR

### CODE REVIEW

**Review by ManojBudumuru-vungle — APPROVED** [review]
Method name and comment updates LGTM

File: adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleNativeAd.h [hunk 1]
```diff
@@ -20,8 +20,12 @@
 /// Asks the receiver to render the ad configuration.
 - (nonnull instancetype)initNativeAdForAdConfiguration:(nonnull GADMediationNativeAdConfiguration *)adConfiguration
                                      completionHandler:(nonnull GADMediationNativeLoadCompletionHandler)completionHandler;
+
+/// Constructor is unavailable. Please use initWithAdConfiguration:completionHandler:.
 - (nonnull instancetype)init NS_UNAVAILABLE;
-- (void)requestAd;
+
+/// Requests a rewarded ad from Vungle.
```
Comment (YueVungle) [inline adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleNativeAd.h:27]: A little confusing. Do we request a native ad or a rewarded ad from Vungle?  I saw the class name is `VungleNativeAd`. Thanks a lot. :)

Comment (stanley-vungle) [reply adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleNativeAd.h:27]: Yes. I copied the comment and missing changing this one. Thanks for catching.

**Review by YueVungle — APPROVED** [review]
LGTM :)

**Pull request merged**